net-misc/teamviewer/teamviewer-15.19.3-r2.ebuild

```diff
@@ -106,13 +106,13 @@ src_install() {
 		newicon -s ${size} tv_bin/desktop/teamviewer_${size}.png teamviewer.png
 	done
 
-	dodoc -r doc
+	dodoc -r doc/*
 
 	# Make docs available in expected location
-	dosym ../../usr/share/doc/${PF}/doc ${dst}/doc
+	dosym ../../usr/share/doc/${PF}/ ${dst}/doc
 
 	# We need to keep docs uncompressed, bug #778617
-	docompress -x /usr/share/doc/${PF}/*
```

Comment:
This is the only problem but it should become "/.".

Reply:
Yes, that works.
Resolved.